### DISCUSSION TITLE
Discussion/Comments

### DISCUSSION THREAD

**Discussion by ElectricRCAircraftGuy** [ElectricRCAircraftGuy/Banned_Google_Reviews#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by murrayzz1** [comment]
There is an easier method of finding banned reviews. If you find all your reviews on a logged in page and an incognito page, you can just scroll through them all. The ones that are on the logged in page but missing from the incognito page are the banned ones.
